scm/src/CCPP_typedefs.F90

```diff
@@ -1277,6 +1280,7 @@ subroutine gfs_interstitial_phys_reset (Interstitial, Model)
     Interstitial%stress_land     = Model%huge
     Interstitial%stress_water    = Model%huge
     Interstitial%theta           = clear_val
+    Interstitial%tkeh            = 0
```

Comment:
Should this be set to `clear_val` or `Model%huge`? Seems that this section sets the variable to bad values so it is clear if they are used but haven't been initialized yet.